lms/templates/ccx/coach_dashboard.html

```diff
@@ -154,5 +159,31 @@
     if ($('#ccx_std_list_messages').length) {
       $('#ccx_std_list_messages')[0].focus();
     }
+    $('#testForm').on('submit', function() {
+
+    });
   });
+  function validateForm(form) {
+    var newCCXName = $(form).find('#ccx_name').val();
+    var $errorMessage =  $('#ccx-create-message');
+    var hasCcxConnector = ${"true" if course and hasattr(course, 'ccx_connector') and course.ccx_connector else "false"};
+
+    if ((!newCCXName || newCCXName === "") && !hasCcxConnector) {
+        $errorMessage.text("${_('Please enter a valid CCX name.')}");
+        $errorMessage.show();
+        console.log("error");
+        return false;
+    } else if (hasCcxConnector) {
+      $errorMessage.html('${_("Please use CCXCon url: {CCXConUrlStart} {url} {CCXConUrlEnd} to create ccx.").format(
+        CCXConUrlStart='<a href="{url}">'.format(url=course.ccx_connector),
+        url=course.ccx_connector, CCXConUrlEnd="</a>"
```

Comment:
Let's change this error message because the CCXCon URL doesn't actually help the user & there's not point in linking to it.  

```
"You cannot create a CCX based on this course because the Advanced Setting "CCX Connector URL" set to "{url}". Contact a course admin to give you access."
```

I think we'll need some translator comments here too.